README.md

```diff
@@ -236,25 +236,26 @@ Linux wheels are built in the [`manylinux1` docker images](https://github.com/py
 - The project directory is mounted in the running Docker instance as `/project`, the output directory for the wheels as `/output`. In general, this is handled transparently by `cibuildwheel`. For a more finegrained level of control however, the root of the host file system is mounted as `/host`, allowing for example to access shared files, caches, etc. on the host file system.  Note that this is not available on CircleCI due to their Docker policies.
 - Alternative dockers images can be specified with the `CIBW_MANYLINUX1_X86_64_IMAGE` and `CIBW_MANYLINUX1_I686_IMAGE` options to allow for a custom, preconfigured build environment for the Linux builds. See [below](#options) for more details.
 
-Moden C++ standards
--------------------
-Python 2.7 use `register` keyword which is prohibited in c++17 standard. 
+### Modern C++ standards
 
-Linux
-=====
-For `manulinux1` images only c++11 standards is supported. c++14 and c++17 will be supported with manylinux2010 image
+Building Python wheels with modern C++ (C++11+) requires a few tricks.
 
-MacOS
-=====
-To get c++11 and c++14 upport need to set `MACOSX_DEPLOYMENT_TARGET` variable to `10.9` value
+Python 2.7 wheels cannot use C++17 at all, because Python 2.7 uses the `register` keyword, which is prohibited in C++17 standard.
 
-To get c++17 support need to set `MACOSX_DEPLOYMENT_TARGET` variable to `10.13` or `10.14` value. `10.13` supports c++17 partialy, eg. filesystem header is in experimental: `#include<filesystem> -> #include<experimental/filesystem>`
+#### Linux
 
-Windows
-=======
-Visual C++ for Python 2.7 do not support modern standards of c++.  
-Also MSVC 10.0 
+For `manylinux1` images, only C++11 standards is supported. C++14 and C++17 will be supported with manylinux2010 image (coming soon!).
 
+#### MacOS
+
+To get C++11 and C++14 support, set `MACOSX_DEPLOYMENT_TARGET` variable to `10.9`.
+
+To get C++17 support need to set `MACOSX_DEPLOYMENT_TARGET` variable to `10.13` or `10.14`. (`10.13` supports C++17 partially, e.g. filesystem header is in experimental: `#include <filesystem>` -> `#include <experimental/filesystem>`)
+
+#### Windows
+
+Visual C++ for Python 2.7 do not support modern standards of C++.
+When building on Appveyor, you'll need to use the 'Visual Studio 2019' image. Note that Python 2.7 isn't supported on this image - you should skip it using `CIBW_SKIP=cp27-win*`.
```

Comment:
For c++14 and c++17 both images should work: 'Visual Studio 2017' or 'Visual Studio 2019'

Reply:
```suggestion
When building on Appveyor, you'll need to use either the 'Visual Studio 2017' or 'Visual Studio 2019' image. Note that Python 2.7 isn't supported on these images - you should skip it using `CIBW_SKIP=cp27-win*`.
```